tools/provision

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+if [ "$EUID" -eq 0 ]; then
+    echo "Error: The provision script must not be run as root" >&2
+    exit 1
+fi
+
+#Make the script independent of the location from where it is
+#executed
+PARENT_PATH=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )
+cd "$PARENT_PATH"
+mkdir -p ../var/log
+LOG_PATH="../var/log/zulip_provision.log"
+PROVISION_PATH="lib/provision_vm.py"
+
+"python" "$PROVISION_PATH" $@ 2>&1 | tee -a $LOG_PATH
```

Comment:
Since provision is run many times, we should probably enter some clear marking of provision starting into the log file.